docs/resources/vw_impala.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
You need to add a template file, similar to: https://github.com/cloudera/terraform-provider-cdp/blob/main/templates/resources/dw_vw_hive.md.tmpl

and generate the docs like that

Reply:
done !

Reply:
The file is not added here, you still haven't add a template: https://github.com/cloudera/terraform-provider-cdp/tree/main/templates/resources